### PR TITLE
[Model] Implement GRADE

### PR DESCRIPTION
**Pull request by Frankshal**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

**Comment by Frankshal** [comment]
现在的问题就是指标对不上，F1MA比原来的高不少，Mean又低不少。但是因为dgl和gammagl数据集加载进来后节点顺序不同，导致测试集的划分不同，这对后面性能评估影响还挺大（试过如果用random shuffle的方法来划分测试集，影响很大）。所以我也不清楚这个结果是我代码出的问题还是数据集自己的问题。

File: gammagl/models/grade.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import GCNConv
+import numpy as np
+
+
+# Multi-layer Graph Convolutional Networks
+class GCN(tlx.nn.Module):
```
Comment (dddg617) [inline gammagl/models/grade.py:7]: We already have GCN models in gammagl.models, so is this necessary?

File: gammagl/models/grade.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        self.act_fn=act_fn
+
+
+    def forward(self, feat,edge_index):
+        for i in range(self.num_layers):
+            feat=self.act_fn(self.convs[i](feat,edge_index))
+        return feat
+
+# Multi-layer(2-layer) Perceptron
+class MLP(tlx.nn.Module):
```
Comment (dddg617) [inline gammagl/models/grade.py:28]: Ditto, we already have MLP module.

File: gammagl/models/grade.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        # init = tlx.nn.initializers.HeNormal(a=math.sqrt(5))
+        self.fc1 = tlx.nn.Linear(in_features=in_feat, out_features=out_feat) #, W_init=init)
+        self.fc2 = tlx.nn.Linear(in_features=out_feat, out_features=in_feat) # , W_init=init)
+
+    def forward(self, x):
+        x = tlx.elu(self.fc1(x))
+        return self.fc2(x)
+
+class GRADE(tlx.nn.Module):
+    r"""
```
Comment (dddg617) [inline gammagl/models/grade.py:40]: Remember to add more detailed rst document. The reference is [here](https://docs.qq.com/pdf/DUXRTTU9tUnB1WnFB?).

File: gammagl/models/grade.py (hunk 1 above)
Comment (Frankshal) [reply gammagl/models/grade.py:7]: It can be implemented by GCNModel instead. However, in the original paper, the activation function used by the author on the Computers dataset is prelu and  GCNModel only has relu for activation function. So if we use GCNModel directly, the model will be slightly different from the original model.
In fact, the GRADE model has the same network structure as GRACE. The only difference is that GRADE uses a more complex  graph augmentation method . So my GRADE model is basically a copy of the GRACE model.

File: gammagl/models/grade.py (hunk 3 above)
Comment (Frankshal) [reply gammagl/models/grade.py]: I have added more details, please check my changes.

File: gammagl/models/grade.py (hunk 2 above)
Comment (Frankshal) [reply gammagl/models/grade.py:28]: The GRADE model has the same network structure as GRACE，so I try to write it as the  same as GRACE. If it needs to be modified, I can replace it with MLP module and the result will be the same.

File: gammagl/models/grade.py (hunk 1 above)
Comment (dddg617) [reply gammagl/models/grade.py:7]: OK, thanks for the work.
